.golangci.yml

```diff
@@ -27,13 +27,12 @@ linters:
     - misspell
     - govet
     - ineffassign
-    - deadcode
```

Comment:
this change is because of

```
WARN [runner] The linter 'varcheck' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused. 
WARN [runner] The linter 'deadcode' is deprecated (since v1.49.0) due to: The owner seems to have abandoned the linter. Replaced by unused.
```